pkg/backend/httpstate/stack.go

```diff
@@ -64,7 +63,7 @@ func (c cloudBackendReference) String() string {
 	// If the project names match, we can elide them.
 	if currentProject != nil && c.project == tokens.Name(currentProject.Name) {
 		// Elide owner too, if it is the default owner.
-		defaultOrg, err := pkgWorkspace.GetBackendConfigDefaultOrg(currentProject)
+		defaultOrg, err := GetDefaultOrg(context.Background(), c.b, c.b.currentProject)
```

Comment:
Don't use context.Background except in top-level functions, if there's no context value to use here use context.TODO()